### PR TITLE
Time Series Use-cases integrated with UI

### PR DESCRIPTION
**Pull request by vishnu-dev**
# What is this PR/ Why we need it?
Tine series UI Integration

# Does the commit involve changes to code / documentation?
(If both check both the boxes)
- [X] Code
  - [X] I have tested the entire code and it works fine :)
  - [ ] I have tested the entire code but there are bugs and I have reported them/ raised an issue for the same :|
  - [ ] No, I have not tested :(
- [ ] Documentation

## Concerned team/ people:
- [ ] Team 1
- [ ] Team 2
- [ ] Team 3
- [ ] Team 4
- [ ] Team 5
- [X] Team 6
- [X] Team 7
- [X] Team 8
- [ ] Team 9

### CODE REVIEW

File: test_tab.py [hunk 1]
```diff
@@ -34,7 +40,7 @@
 from ac_classes import BiDataModel as bdmodel
 from ac_classes import MultiDataModel as mdmodel
 from nonlinear_regression import NonLinearRegression as nlr
-from anova import main as anova_main
+# from anova import main as anova_main
```
Comment (kn-neelalohitha) [inline test_tab.py:43]: Please un-comment this line 43

File: test_tab.py [hunk 2]
```diff
@@ -12,15 +12,21 @@
...
+from arima.runner import init_ar, check_seasonality, do_diff, predict_using_arima
+
 matplotlib.use("TkAgg")
 from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
 from matplotlib.figure import Figure
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 from tabulate import tabulate   # for table
 import pandas as pd
-import seaborn as sns
```
Comment (kn-neelalohitha) [inline test_tab.py:23]: Please include this line as well 
```python
 import seaborn as sns
```
we use it for multiple linear regression

File: test_tab.py (hunk 1 above)
Comment (kn-neelalohitha) [reply test_tab.py:43]: We still use anova_main [here](https://github.com/AnalyticsCal/AnalyticsCal-Classroom/blob/d54333b1a52a50fc5a1c2a59a694d58dd6178034/test_tab.py#L819)

File: anova - Copy.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
+# How to use..?
```
Comment (kn-neelalohitha) [inline anova - Copy.py]: Rename this file `anova.py`

File: anova.py [hunk 4]
```diff
@@ -1,117 +1,58 @@
 # How to use..?
 # 1. import anova
-# 2. anova_dictionary = anova.main(x_values_list, y_values_list, coefficients_array)
-# NOTE: coefficients_array should be in the order of beta0, beta1, beta2 ...
-# For example equation = 11x - 20
-# coefficients_array = [-20, 11], x_values_list = (2, 4, 6, 8, 10), y_values_list = (6, 18, 38, 66, 102)
+# 2. Pass y_list=y vaues, y_cap_list=y predected values, degree_of_freedom= number of parameters to class
```
Comment (kn-neelalohitha) [inline anova.py:3]: Please rename this file as `anova_class.py`

**Review by kn-neelalohitha — CHANGES_REQUESTED** [review]
@vishnu-dev thanks for integrating! Please go through review comments and include these small changes also!
Or else everything is fine!

**Comment by kn-neelalohitha** [comment]
Requested changes will be incorporated. Everything else is awesome!
LGTM!
Thanks @vishnu-dev !!

**Review by kn-neelalohitha — APPROVED** [review]

**Comment by vishnu-dev** [comment]
@kn-neelalohitha Thanks for fixing the conflicts.

**Comment by kn-neelalohitha** [comment]
@vishnu-dev It was a pleasure working with u! And thank u for the feature!

**Pull request merged**